Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -1093,13 +1116,15 @@ - (CGSize)cellSizeForViewWidth:(int)viewWidth contentWidth:(int)contentWidth
 
     cellSize.height += self.dateHeaderHeight;
     cellSize.height += self.footerHeight;
+    if (self.hasTapForMore) {
+        cellSize.height += self.tapForMoreHeight;
+    }
 
     if (self.shouldHaveFailedSendBadge) {
         cellSize.width += self.failedSendBadgeSize;
     }
 
-    cellSize.width = ceil(cellSize.width);
-    cellSize.height = ceil(cellSize.height);
+    cellSize = CGSizeCeil(cellSize);
```

Comment:
Nice, didn't know about this method.

Reply:
It's custom - see above.  Usually on obj-c projects I end up with a header full of these kinds of custom macros.  They make CG code way more readable.